code/common/subscriptions.q

```diff
@@ -82,6 +81,8 @@ subscribe:{[tabs;instrs;setschema;replaylog;proc]
 			@[{-11!x;};details 1;{.lg.e[`subscribe;"could not replay the log file: ", x]}];
 			/-reset the upd function back to original upd
 			@[`.;`upd;:;origupd]];
+		if[replaylog&null details[1;1];
+			.lg.w[`subscribe;"replaylog set to true but TP not using log file"]];
```

Comment:
Could you just make this an error? Warnings tend to get ignored, with error the process will fail and then someone (a developer) will fix it whichever way they want it (to replay or not to replay)